src/uiSelectSortDirective.js

```diff
@@ -92,6 +92,8 @@ uis.directive('uiSelectSort', ['$timeout', 'uiSelectConfig', 'uiSelectMinErr', f
         }
 
         move.apply(theList, [droppedItemIndex, newIndex]);
+        
+        scope.$parent.$selectMultiple.updateModel();
```

Comment:
I've just tried this and the 2nd controller is `undefined`? I'm not quite sure what to try...